### PR TITLE
Test against more environments

### PR DESCRIPTION
**Pull request by mattbriancon**

### CODE REVIEW

**Comment by ezarowny** [comment]
Thanks for the PR! I'll take a look at this.

**Comment by mattbriancon** [comment]
Thanks for the quick response. I just threw in the `-Wall` out of curiosity but I'm happy to remove.

**Comment by ezarowny** [comment]
I'm at PyCon right now. It seemed like appropriate time to look at this 👏

**Comment by ezarowny** [comment]
One more thing with Django: I've purposefully left out certain versions of Python from certain Django releases: https://docs.djangoproject.com/en/1.11/faq/install/#what-python-version-can-i-use-with-django

**Comment by ezarowny** [comment]
Also, I don't think 1.11.2 has been released yet?

**Comment by mattbriancon** [comment]
You're right, I didn't catch the `under development` note in the 1.11.2 release notes. Updated all of the Django versions to the latest patch. I'd like to get this testing against Django and Flask master too (under `allow_failures`) but I don't have enough time this afternoon.

Let me know if there are anything other changes I can make. Enjoy PyCon!

**Comment by ezarowny** [comment]
Testing against master is a pretty good idea! Good to see if any breaking changes are coming in the pipeline.

File: .travis.yml [hunk 1]
```diff
@@ -29,40 +43,66 @@ matrix:
...
+      env: DJANGO_VERSION=1.11.1
 
 install:
   - if [ -v FLASK_VERSION ]; then pip install Flask==$FLASK_VERSION; fi
   - if [ -v TWISTED_VERSION ]; then pip install Twisted==$TWISTED_VERSION treq; fi
   - if [ -v DJANGO_VERSION ]; then pip install Django==$DJANGO_VERSION; fi
 
 script:
-  - python setup.py test
+  - python -Wall setup.py test
```
Comment (ezarowny) [inline .travis.yml]: Eh, let's strip this out for now. I'll look into this a bit later though I would wager that the tests would pass just fine with it there.

File: .travis.yml [hunk 2]
```diff
@@ -29,40 +43,66 @@ matrix:
...
+      env: DJANGO_VERSION=1.10.7
+    - python: "3.5"
+      env: DJANGO_VERSION=1.11.1
+
+    - python: "3.6"
+      env: DJANGO_VERSION=1.8.18
+    - python: "3.6"
+      env: DJANGO_VERSION=1.9.13
+    - python: "3.6"
+      env: DJANGO_VERSION=1.10.7
```
Comment (ezarowny) [inline .travis.yml]: Python 3.6 is only officially supported in Django 1.11+

**Comment by ezarowny** [comment]
Changes look good! Thanks for the quick updates. I'll merge this once the tests pass. 👍

**Pull request merged**